apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx

```diff
@@ -57,13 +59,16 @@ export const CreateTicketForm: React.FC = () => {
     const { organization, link } = useOrganization()
     const router = useRouter()
     const auth = useAuth() as { user: { id: string } }
+    const client = useApolloClient()
+    const { addTicketToQueryCacheForTicketCardList } = useCacheUtils(client.cache)
 
     const action = Ticket.useCreate(
         {
             status: OPEN_STATUS,
             source: DEFAULT_TICKET_SOURCE_CALL_ID,
         },
-        () => {
+        (ticket) => {
+            addTicketToQueryCacheForTicketCardList(ticket)
```

Comment:
As I see, the solution does not solve problem, if ticket was created by user A, while we're on user B account from same organization, am I right?

May be then disabling cache in `/contact/[id]/index` ticket query was "goto solution"?

Reply:
> As I see, the solution does not solve problem, if ticket was created by user A, while we're on user B account from same organization, am I right?
The solution modifies query cache used at particular contact page.
![image](https://user-images.githubusercontent.com/25384290/149707697-a3bba716-d314-490d-895c-00447aad997d.png)
> May be then disabling cache in /contact/[id]/index ticket query was "goto solution"?
If you mean this cache setting
![image](https://user-images.githubusercontent.com/25384290/149707899-de5b8271-178e-454b-9a3d-147831cc33b6.png)
I thought about this, but I try to use cache in right way to prevent server overloading. Let's say, disabling of cache is the fall back decision.